# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/loader/mozJSSubScriptLoader.cpp
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c js/xpconnect/loader/mozJSSubScriptLoader.cpp
--- a/js/xpconnect/loader/mozJSSubScriptLoader.cpp	Tue Feb 12 17:58:19 2019 +0100
+++ b/js/xpconnect/loader/mozJSSubScriptLoader.cpp	Tue Feb 12 16:08:25 2019 +0000
@@ -436,7 +436,7 @@
   NS_ENSURE_SUCCESS(rv, rv);
 
   nsCOMPtr<nsIStreamListener> listener = loader.get();
-  return channel->AsyncOpen2(listener);
+  return channel->AsyncOpen(listener);
 }
 
 bool mozJSSubScriptLoader::ReadScript(nsIURI* uri, JSContext* cx,
@@ -463,7 +463,7 @@
 
   if (NS_SUCCEEDED(rv)) {
     chan->SetContentType(NS_LITERAL_CSTRING("application/javascript"));
-    rv = chan->Open2(getter_AddRefs(instream));
+    rv = chan->Open(getter_AddRefs(instream));
   }
 
   if (NS_FAILED(rv)) {